aiohttp/web_urldispatcher.py

```diff
@@ -38,7 +38,13 @@
 
 
 class RouteDef(namedtuple('_RouteDef', 'method, path, handler, kwargs')):
-    # TODO: add __repr__
+    def __repr__(self):
+        info = []
+        for name, value in sorted(self.kwargs.items()):
+            info += ", {}={}".format(name, value)
+        return ("<RouteDef {method} {path} -> {handler.__name__!r}"
+                "{info}>".format(method=self.method, path=self.path,
+                                 handler=self.handler, info=''.join(info)))
```

Comment:
i think will be better if info will be string 
then you can delete ''.join(info)